### PR TITLE
add support for SSL CA cert files

### PR DESCRIPTION
**Pull request by dpassante**
##### SUMMARY

Cloudstack modules don't pass the `verify` config item to the cs module.
It makes them fail when the Cloudstack endpoint requires a CA cert file.

This PR add support for `CLOUDSTACK_VERIFY` environment variable and for the cloudstack.ini `verify` option.

For consistency, I also added the new common parameter 'api_ssl_verify' which allows to specify the CA cert file path as a parameter of the modules.

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Comment by dpassante** [comment]
CI is very nice!
I will try to find a way to test this new param.

**Comment by resmo** [comment]
yeah, but I doubt these coverage numbers are correct.

File: plugins/module_utils/cloudstack.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def cs_argument_spec():
         api_http_method=dict(choices=['get', 'post'], default=os.environ.get('CLOUDSTACK_METHOD')),
         api_timeout=dict(type='int', default=os.environ.get('CLOUDSTACK_TIMEOUT')),
         api_region=dict(default=os.environ.get('CLOUDSTACK_REGION') or 'cloudstack'),
+        api_ssl_verify=dict(default=os.environ.get('CLOUDSTACK_VERIFY')),
```
Comment (resmo) [inline plugins/module_utils/cloudstack.py:38]: hmm `ssl_verify` is IMHO misleading: mind changing to `api_verify_ssl_cert`?

File: plugins/doc_fragments/cloudstack.py [hunk 2]
```diff
@@ -51,6 +51,13 @@ class ModuleDocFragment(object):
...
+  api_ssl_verify:
+    description:
+      - CA authority cert file.
+      - If not given, the C(CLOUDSTACK_VERIFY) env variable is considered.
+      - As the last option, the value is taken from the ini config file, also see the notes.
+      - Fallback value is C(null) if not specified.
+    type: str
 requirements:
   - python >= 2.6
   - cs >= 0.6.10
```
Comment (resmo) [inline plugins/doc_fragments/cloudstack.py:63]: I guess, we need to increase the cs version as well?

**Comment by resmo** [comment]
I just checked the line coverage again in codecov.io and it looks like `84.35%`, coverage is still the same which is what I would have expected. So this LGTM

File: plugins/doc_fragments/cloudstack.py (hunk 2 above)
Comment (dpassante) [reply plugins/doc_fragments/cloudstack.py]: Right, it seems the option appeared in version 0.9.0.

File: plugins/module_utils/cloudstack.py (hunk 1 above)
Comment (dpassante) [reply plugins/module_utils/cloudstack.py]: OK, done.

**Comment by dpassante** [comment]
Fine! I have no idea how to validate this in the integration tests :)

**Comment by dpassante** [comment]
Since the parameter has no effect on a non-SSL API, I don't see any relevant tests to add to the integration tests.

ready_for_review

**Comment by resmo** [comment]
Thanks

**Pull request merged**